src/http/server/hyper_conn.rs

```diff
@@ -5,7 +5,6 @@ use crate::service::Service;
 use crate::service::{Context, HyperService};
 use crate::stream::Stream;
 use crate::tcp::utils::is_connection_error;
-use futures::FutureExt;
```

Comment:
Deleting this is incorrect, you no longer `::fuse()` which is going to be ok for the tests, but in production in can mean a panic.

Please use `futures_lite::stream::StreamExt`, so that you can re-add the `.fuse()` calls below @dalprahcd

Reply:
Understood, but I may need help with this task. The `.fuse()` method is defined for `Streams`, but `guard.cancelled()` is a `Future`, so I am having a hard time using it.

Reply:
Seems that they do not have such a fuse functionality for futures.
All good. I'll make something myself in a follow-up MR. This one is good to go then.

Thanks for the contribution!